lib/src/react_client/synthetic_event_factories.dart

```diff
@@ -0,0 +1,325 @@
+import 'dart:html';
```

Comment:
I would vote for `event_factory.dart` to go along with the suggestion for `component_factory.dart`.